cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSinkTest.java

```diff
@@ -1533,76 +1615,101 @@ public void testNativeTypeRowBatchCsvNotMetadata() throws CygnusBadConfiguration
 
     @Test
     public void testNativeTypeRowBatchJson() throws CygnusBadConfiguration, CygnusRuntimeError, CygnusPersistenceError, CygnusBadContextData {
-        String attr_native_types = "true";
-        NGSIHDFSSink ngsihdfsSink= new NGSIHDFSSink();
-        ngsihdfsSink.configure(createContextforNativeTypes("json-row", null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, attr_native_types, "true"));
         NGSIBatch batch = prepaireBatch();
         String destination = "someDestination";
+        String file_format = "json-row";
+        NGSIHDFSSink ngsihdfsSink = new NGSIHDFSSink();
+        ngsihdfsSink.configure(createContextforNativeTypes(null, null, null, null, null, null, null, null, null, null, null, file_format, null, null, null, null, null, null, null, null, null, null));
         try {
             batch.startIterator();
+            //NGSIGenericAggregator aggregator = new NGSIGenericColumnAggregator();
```

Comment:
leftover?

Reply:
Fixed in [bf42de0](https://github.com/telefonicaid/fiware-cygnus/pull/1818/commits/bf42de008c59fb41d740ec6b851aee434e50fb7b)